tests/test_clinvar.py

```diff
@@ -38,7 +38,7 @@ def test_observed_pubmed_refs(self):
         self.assertEqual(self.test_clinvar_record.observed_pubmed_refs, [11524702, 12145752])
 
     def test_clinical_significance(self):
-        self.assertEqual(self.test_clinvar_record.clinical_significance, "pathogenic")
+        self.assertEqual(self.test_clinvar_record.clinical_significance, "Pathogenic")
```

Comment:
Does capitalization really matter? Can't we work with lower case strings the whole time?